src/commands/testing/spec/testing.spec.server.ts

```diff
@@ -26,6 +26,11 @@ import { getConstants } from '../../../constants'
 
 describe('sasjs test', () => {
   let target: Target
+  const testUrl = (test: string) =>
+    `https://sas.analytium.co.uk/SASJobExecution/?_program=/Public/app/cli-tests/${
```

Comment:
perhaps this should be a global variable? So other teams can run tests on other platforms